app/components/form/fields/useItemsList.ts

```diff
@@ -42,3 +42,14 @@ export const useCustomRouterItems = () => {
 
   return { isLoading, items: routerItems }
 }
+
+// Get a list of subnets for a given VPC; If no project/VPC is provided, use the VPC from the VpcSelector
+export const useVpcSubnets = ({ project, vpc }: { project?: string; vpc?: string }) => {
```

Comment:
Might as well make these args required since they're passed in at both call sites